### ISSUE TITLE
vmalert sends alerts for pending rules and triggers send_resolve condition

### ISSUE THREAD

**Issue by f41gh7** [VictoriaMetrics/VictoriaMetrics#483]
**Describe the bug**

 VmAlert sends alerts with inactive status (even if previous state was pending) to alert manager with EndTime = StartTime. So alertmanager counts it as resovled event and send resolved notification to webhook.

**To Reproduce**
1) enable at alertmanager send_resolve for alertmanager receiver
```
        webhook_configs:
            - send_resolved: true
```

2) create simple rule:
```
groups:
  - name: groupGorSingleAlert
    rules:
      - alert: VMRows
        for: 10m
        expr: rate(vmalert_remotewrite_errors_total[1m]) == 0
        annotations:
          description: "error writing to remote writer from vmalert {{$value}}"
```
3) start vmalert with rule and datasource, check alert status:
```
 curl localhost:8880/api/v1/alerts
{"data":{"alerts":[{"id":"15451015430077619995","name":"VMRows","group_id":"9641763344171447408","expression":"rate(vmalert_remotewrite_errors_total[1m]) == 0","state":"pending"...
```
4) change rule to  inactive state
```

groups:
  - name: groupGorSingleAlert
    rules:
      - alert: VMRows
        for: 10m
        expr: rate(vmalert_remotewrite_errors_total[1m]) < 0
        annotations:
          description: "error writing to remote writer from vmalert {{$value}}"
```

4) reload rules
```
curl localhost:8880/-/reload
```
5) check alertmanager for resolved event (with tcpdump or at your notification system).

**Expected behavior**
Send resolved events only for previously fired rules or do not send inactive events at all, it`s work for alertmanager to count it as resolved after endTime.

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./vmalert-prod --version
vmalert-20200512-200621-tags-v1.35.4-0-g96e001d2
```

**Used command-line flags**
-evaluationInterval=5s

**Comment by hagen1778** [comment]
Thanks for report @f41gh7 !

**Comment by hagen1778** [comment]
Hi @f41gh7 
The fix was added in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/92212f04da8e6fdc46bc6bdce7f9913cc17b83d3. Could you pls verify the fix by building `vmalert` from the mentioned commit?

**Comment by f41gh7** [comment]
looks good, thx!

**Comment by valyala** [comment]
FYI, the bugfix has been included in the release [v1.35.6](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
